huggingface-gpt2/ort_addon/ort_supplement/docker/transformers-ort-gpu/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM mcr.microsoft.com/azureml/onnxruntime-training:0.1-rc1-openmpi4.0-cuda10.1-cudnn7.6-nccl2.4.8
```

Comment:
`:latest` tag might help to avoid touching this sample when we refresh the base images.

Reply:
We use a --pull command in docker build that pulls in the latest base image.